lxd/instance/drivers/driver_qemu.go

```diff
@@ -2007,7 +2007,7 @@ func (d *qemu) qemuArchConfig(arch int) (path string, bus string, err error) {
 		}
 
 		return path, "pcie", nil
-	} else if arch == osarch.ARCH_64BIT_ARMV8_LITTLE_ENDIAN {
+	} else if arch == osarch.ARCH_32BIT_ARMV7_LITTLE_ENDIAN || arch == osarch.ARCH_32BIT_ARMV8_LITTLE_ENDIAN || arch == osarch.ARCH_64BIT_ARMV8_LITTLE_ENDIAN {
```

Comment:
Actually, could that be a case for `slices.Contains()`?

Reply:
This could be a case for `slices.Contains()`. Personally, I prefer using logical OR for readability with a smaller number of options, but we could create a slice of supported ARM architectures and check using `slices.Contains()`.

It would look something like:

```Go
supportedArchs := []int{
    osarch.ARCH_32BIT_ARMV7_LITTLE_ENDIAN,
    osarch.ARCH_32BIT_ARMV8_LITTLE_ENDIAN,
    osarch.ARCH_64BIT_ARMV8_LITTLE_ENDIAN,
}
...
else if slices.Contains(supportedArchs, arch) {
...
}
```

The `supportedArchs` slice could be located in `driver_qemu.go` or `architectures.go` (with a "getter" function).

Reply:
Elsewhere in the code I've seen the slice inlined as in:
```
else if slices.Contains([]int{osarch.ARCH_32BIT_ARMV7_LITTLE_ENDIAN, osarch.ARCH_32BIT_ARMV8_LITTLE_ENDIAN, osarch.ARCH_64BIT_ARMV8_LITTLE_ENDIAN}, arch) {
```

Really a question of style... inspired by my little Python background where `if arch in ("osarch.ARCH_32BIT_ARMV7_LITTLE_ENDIAN", ... ""):` would have worked ;)

Reply:
Thanks for sharing that one-liner! I'm happy with any of the above approaches. I'll wait for @tomponline to comment.

Reply:
I'm fine with it as it is.

But if we were to use that logic in multiple places then I would say we should have a function that answers the specific question/classifier, e.g. `architectureSupportsUEFI` rather than returning a slice.